pulsar-common/src/main/java/org/apache/pulsar/common/api/Commands.java

```diff
@@ -891,4 +891,30 @@ private static int getCurrentProtocolVersion() {
         Crc32c,
         None;
     }
+
+    public static ByteBuf newPartitionMetadataRequest(String topic, long requestId, String clientAuthRole) {
```

Comment:
We should reuse the same implementation, either passing `null` or having an ovverride method and passing null